R/setWatermark.R

```diff
@@ -0,0 +1,40 @@
+#' @title setWatermark
+#' @param plotHandle ggplot object
+#' @param label to be written in watermark (class Label and character allowed)
+#' @param angle (optional argument) angle of the watermark
+#' @return plotHandle ggplot object with watermark background as a grob
+#' @description
+#' setWatermark set watermark as defined by label
+#' @import  ggplot2

```

Comment:
This is required at least once somewhere to ensure the package is loaded. Might not be required if we declare it as a dependencies. Works like that for now